testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -145,7 +145,10 @@ def _wait_for_containers(self, expected_containers):
                 "docker-compose -p %s %s ps -q" % (self.name, files_args), shell=True
             )
             running_countainers_count = len(out.split())
-            if running_countainers_count == expected_containers:
+            if running_countainers_count in (
+                expected_containers,
+                expected_containers + 1,
+            ):
```

Comment:
Why not?

Reply:
because you break other tests (migrate os to ent) because of an extra container running (I suppose it is your session scope client).

Reply:
I don't understand why this would break other tests. The `NUM_CONTAINERS` variables have been updated, and the service is added to `BASE_FILES` so it should be included in all tests. I think the lines above are the wrong fix, because we are just relaxing conditions, but the number of containers should be exact.

Reply:
@kacf it did break those tests, though. Look here: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/236766853

Reply:
Hmm, ok. I'll investigate a bit more.

Reply:
Doh, of course. I need to adjust `NUM_CONTAINERS` by 2, not 1, since `docker-compose.connect.yml` also contains the nats service.

I'll fix that and update it in my original https://github.com/mendersoftware/integration/pull/1120, along with your new commits. Closing this.